src/frontend/src/components/NavBar/index.tsx

```diff
@@ -1,98 +1,24 @@
-import { FEATURE_FLAGS, isFlagOn } from "src/components/Split";
-import { InviteModal } from "src/views/GroupMembersPage/components/MembersTab/components/InviteModal";
-import { GroupDetailsDropdown } from "./components/GroupDetailsDropdown";
-import RightNav from "./components/RightNav";
-import {
-  DropdownClickTarget,
-  LeftNav,
-  Logo,
-  LogoAnchor,
-  NavBar,
-  NavOrg,
-  Separator,
-  StyledIcon,
-} from "./style";
-
-const NavBarLoggedIn = (): JSX.Element => {
-  const [anchorEl, setAnchorEl] = useState<Element | null>(null);
-  const [isGroupDetailsDropdownOpen, setIsGroupDetailsDropdownOpen] =
-    useState<boolean>(false);
-  const [isInviteModalOpen, setIsInviteModalOpen] = useState<boolean>(false);
-  const { data: userInfo } = useUserInfo();
-
-  const group = userInfo?.group;
-  const route = userInfo ? ROUTES.DATA : ROUTES.HOMEPAGE;
-
-  const flag = useTreatments([FEATURE_FLAGS.user_onboarding_v0]);
-  const isUserOnboardingFlagOn = isFlagOn(
-    flag,
-    FEATURE_FLAGS.user_onboarding_v0
-  );
-
-  const toggleDropdown: MouseEventHandler = (e) => {
-    const newAnchor = isGroupDetailsDropdownOpen ? null : e.currentTarget;
-    setAnchorEl(newAnchor);
-    setIsGroupDetailsDropdownOpen(!isGroupDetailsDropdownOpen);
-  };
-
-  const orgElements = (
-    <React.Fragment>
-      <Separator />
-      <NavOrg>{group?.name}</NavOrg>
-    </React.Fragment>
-  );
-
-  function hasOrg(): JSX.Element | null {
-    if (group === undefined) {
-      return null;
-    } else {
-      return orgElements;
-    }
-  }
-
-  const orgSplash = hasOrg();
-  const name = group?.name;
-
-  return (
-    <NavBar data-test-id="navbar">
-      <LeftNav>
-        <Link href={route} passHref>
-          <LogoAnchor href="passHref">
-            <Logo data-test-id="logo" />
-            {!isUserOnboardingFlagOn && orgSplash}
-          </LogoAnchor>
-        </Link>
-        {name && isUserOnboardingFlagOn && (
-          <>
-            <Separator />
-            <InviteModal
-              onClose={() => setIsInviteModalOpen(false)}
-              open={isInviteModalOpen}
-              groupName={name}
-            />
-            <DropdownClickTarget onClick={toggleDropdown}>
-              <NavOrg>{name}</NavOrg>
-              <StyledIcon>
-                <Icon sdsIcon="chevronDown" sdsSize="xs" sdsType="static" />
-              </StyledIcon>
-              <GroupDetailsDropdown
-                anchorEl={anchorEl}
-                onClickInvite={() => setIsInviteModalOpen(true)}
-                open={isGroupDetailsDropdownOpen}
-              />
-            </DropdownClickTarget>
-          </>
-        )}
-      </LeftNav>
-      <RightNav />
-    </NavBar>
+import AppNavBar from "src/components/NavBar/components/AppNavBar";
+import StaticPageNavBar from "src/components/NavBar/components/StaticPageNavBar";
+
+// we show a different nav bar for paths in the app vs not in the app
+const pathsNotInApp: string[] = [
```

Comment:
Should the Resources page be included here? Or are we eventually offloading that to the help center?

Reply:
ah, well then
https://github.com/chanzuckerberg/czgenepi/pull/1187